ssl/ssl_locl.h

```diff
@@ -1079,7 +1087,22 @@ struct ssl_ctx_st {
     /* Callback for SSL async handling */
     SSL_async_callback_fn async_cb;
     void *async_cb_arg;
+#ifndef OPENSSL_NO_QUIC
+    const SSL_QUIC_METHOD *quic_method;
+#endif
+};
+
+#ifndef OPENSSL_NO_QUIC
+struct quic_data_st {
+    struct quic_data_st *next;
+    OSSL_ENCRYPTION_LEVEL level;
+    size_t offset;
```

Comment:
I left this in here, but it's not used. I was thinking of adding another QUIC method (i.e. s->method) that would replace `ssl_read_bytes` with one that would read from these `struct quic_data_st` structures. That is a little more involved, and I didn't get to it. If I were to do that, then the **statem_quic.c** file would no longer be necessary, and `tls_get_message_header()` and `tls_get_message_body()` could be used instead of `quic_get_message()`.

Reply:
Using a QUIC version of SSL_METHOD will not work, so I'm leaving the code as-is.